tests/rules_test.py

```diff
@@ -474,9 +475,32 @@ def test_new_term():
     with mock.patch('elastalert.ruletypes.Elasticsearch') as mock_es:
         mock_es.return_value = mock.Mock()
         mock_es.return_value.search.return_value = mock_res
+        call_args = []
+
+        # search is called with a mutable dict containing timestamps, this is required to test
```

Comment:
A better phrasing would be to call this a callback that you've injected into your mock. But lgtm.